src/index.ts

```diff
@@ -14,6 +15,20 @@ import { rpcError } from './helpers/utils';
 const app = express();
 const PORT = process.env.PORT || 3005;
 
+Sentry.init({
+  dsn: process.env.SENTRY_DSN,
+  integrations: [
+    new Sentry.Integrations.Http({ tracing: true }),
+    new Sentry.Integrations.Express({ app }),
+    ...Sentry.autoDiscoverNodePerformanceMonitoringIntegrations()
```

Comment:
it could send a lot of requests to the sentry (so we can exceed limits for the account) - we need to observe